resources/pipeline/rackhd/ova/stack_config.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
if it's a "ova"'s stack_config, why including others like vagrant and hop stack(it should not be exposed.)

Reply:
OK~ I'll try to only replace the ova section.